lib/litmus/type/date_time.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Litmus.Type.DateTime do
+  @moduledoc """
+  This type validates and converts ISO-8601 strings into `DateTime`s.
```

Comment:
Can we explicitly mention `converts extended “Date and time of day” ISO-8601 strings` ?